perfkitbenchmarker/disk.py

```diff
@@ -21,6 +21,11 @@
 
 from perfkitbenchmarker import resource
 
+STANDARD = 'standard'
+SSD = 'ssd'
```

Comment:
Do you want the constant to be named REMOTE_SSD, the value (that people have to type with --scratch_disk_type) to be remote_ssd, or both?